lib/galaxy/webapps/galaxy/api/tasks.py

```diff
@@ -14,11 +17,12 @@
 
 @router.cbv
 class FastAPITasks:
+    manager: AsyncTasksManager = depends(AsyncTasksManager)  # type: ignore[type-abstract]
```

Comment:
Over and over mypy just sucks on this point - I want to ``--disable-error-code=type-abstract`` this whole project. xref https://github.com/python/mypy/issues/4717#issuecomment-1264388209. I don't think I've ever seen a bug found via that issue - it just makes good code and code patterns look ugly.

Reply:
Yeah, it's a bit disappointing that mypy doesn't handle this nicely...